Jenkinsfile_CNP

```diff
@@ -11,7 +11,7 @@ def channel = '#bsp-build-notices'
 GradleBuilder builder = new GradleBuilder(this, product)
 
 List<LinkedHashMap<String, Object>> secrets = [
-  secret('test-s2s-secret', 'TEST_S2S_SECRET')
+  secret('test-sample-app-s2s-secret', 'TEST_S2S_SECRET')
```

Comment:
is this an s2s secret for service called `bulk-scan-sample-app-tests`?

Reply:
yeah

Reply:
we already used `test-s2s-secret` name for `bulk-scan-processor-tests`. so I had to rename it

Reply:
maybe `bulk-scan-sample-app-tests-s2s-secret` for clarity?

Reply:
renamed. Also, deleted the old secret from vault